.ci/Jenkinsfile

```diff
@@ -36,13 +36,19 @@ pipeline {
         stage('Build nccl-rdma-sharp-plugins') {
             steps {
                 echo 'Building nccl-rdma-sharp-plugins...'
-                sh 'set -o pipefail; ${WORKSPACE}/.ci/build_nccl_rdma_sharp_plugins.sh 2>&1 | tee ${ARTEFACT_DIR}/build_nccl_rdma_sharp_plugins.log'
+                sh """#!/bin/bash
```

Comment:
Don't you need `-eE`? Or maybe also `-l`?

Reply:
`set -o pipefail` is needed specifically for `... | tee ...` to correctly catch script exit code in this shell pipe. Others are unnecessary here (but we may use them inside the scripts we call during Jenkins step.)